.github/workflows/golangci-lint.yml

```diff
@@ -4,35 +4,83 @@ on:
         with:
-          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v${{ steps.tool-versions.outputs.golangci-lint_version }}
-
-          # Optional: working directory, useful for monorepos
+          version: v${{ needs.golangci-lint-version.outputs.version }}
           working-directory: relayer
 
-          # Optional: golangci-lint command line arguments.
-          args: --enable=gofmt --tests=false --exclude-use-default
-
-          # Optional: show only new issues if it's a pull request. The default value is `false`.
-          only-new-issues: true
-
-          # Optional: if set to true then the action don't cache or restore ~/go/pkg.
-          # skip-pkg-cache: true
+  golang_lint_ops:
+    name: Golang Lint Ops
+    runs-on: ubuntu-latest
+    needs: [golangci-lint-version]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Install Nix
+        uses: cachix/install-nix-action@daddc62a2e67d1decb56e028c9fa68344b9b7c2a # v18
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v${{ needs.golangci-lint-version.outputs.version }}
+          working-directory: ops
 
-          # Optional: if set to true then the action don't cache or restore ~/.cache/go-build.
-          # skip-build-cache: true
+  golang_lint_integration_tests:
+    name: Golang Lint Integration Tests
+    runs-on: ubuntu-latest
+    needs: [golangci-lint-version]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Install Nix
+        uses: cachix/install-nix-action@daddc62a2e67d1decb56e028c9fa68344b9b7c2a # v18
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+      - name: Lint integration-tests
+        run: nix develop -c make lint-go-test
+      # Note: I could not figure out why the golangci-lint-action would not work even though it is technically running the same as above, error message is this:
+      # Running [/home/runner/golangci-lint-1.50.1-linux-amd64/golangci-lint run --out-format=github-actions --path-prefix=integration-tests --exclude=dot-imports] in [/home/runner/work/chainlink-starknet/chainlink-starknet/integration-tests] ...
+      # level=warning msg="[runner] Can't run linter goanalysis_metalinter: inspect: failed to load package client: could not load export data: no export data for \"github.com/smartcontractkit/chainlink-testing-framework/client\""
+      # level=error msg="Running error: 1 error occurred:\n\t* can't run linter goanalysis_metalinter: inspect: failed to load package client: could not load export data: no export data for \"github.com/smartcontractkit/chainlink-testing-framework/client\"\n\n"
```

Comment:
I have a guess on this...do you think the reference in `go.mod` to the chainlink-testing-framework was dropped? 

if you check: https://github.com/smartcontractkit/chainlink-testing-framework/tree/d40b562b818b
it shows
>This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

so maybe it's not findable anymore 🤷🏼

Reply:
let me see if i can fix it

Reply:
nevermind - my idea to fix didn't work